src/aerie_cli/__main__.py

```diff
@@ -31,6 +32,15 @@ def print_version(print_version: bool):
         typer.echo(__version__)
         raise typer.Exit()
 
+def set_configuration(configuration: str):
```

Comment:
Suggesting some more specific names to disambiguate:

- `set_alternate_configuration` instead of `set_configuration`, to show that this is in lieu of the active session configuration
- `configuration_identifier` instead of `configuration`, to show that its the name/filename of the configuration